tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -70,3 +72,70 @@ def tap_install_button(self):
 
     def get_app_name(self):
         return self.root_element.find_element(*self._name_locator).text
+
+class MarketplaceDev(Marketplace):
+
+    name = 'Dev'
+    manifest_url = 'https://marketplace.firefox.com/app/bee10fbb-7829-4d99-8a9f-a6d0a905138e/manifest.webapp'
+
+    _enable_addons_locator = (By.ID, 'enable-addons')
+    _m_in_statusbar_locator = (By.CSS_SELECTOR, 'a[href="/addon/m-in-status-bar"]')
+    _addon_install_locator = (By.CSS_SELECTOR, '.addon-install-btn')
+
+    def install_addon(self, addon_name):
```

Comment:
This method is about 60 lines long. At a first glance, it's hard to know what are the steps in order to install an addon. How about
``` python
def install_addon(self, addon_name):
    self._open_searchbox()
    self._search(addon_name)
    self._enable_addons()
    addon_page = self._open_addon()
    addon_page.install
```
?